creation/lib/cvWParamDict.py

```diff
@@ -1364,7 +1364,7 @@ def populate_group_security(client_security, params, sub_params, group_name):
 
                 if (pel["pool_idx_len"] is None) and (pel["pool_idx_list"] is None):
                     # only one
-                    dn = x509Support.extract_DN(proxy_fname)
+                    dn = x509Support.extract_DN(proxy_fname, exit_if_bad=False)
```

Comment:
Put this in a try except or protect it with an if (see x509Support below):
```
                    try:
                        dn = x509Support.extract_DN(proxy_fname)
                    except ValueError:
                        pass
OR
                    if os.path.exists(proxy_fname):
                        dn = x509Support.extract_DN(proxy_fname)
                    else:
                        dn = ""
```

Reply:
put in a try/except as suggested